pkg/controller/machineset/zz_generated.api.register.go

```diff
@@ -118,8 +118,8 @@ func (c *MachineSetController) reconcile(key string) (err error) {
 }
 
 func (c *MachineSetController) Run(stopCh <-chan struct{}) {
+	controller.GetDefaults(c.controller).Run(stopCh)
```

Comment:
Just to confirm my understanding, this is the change you were talking about [here](https://github.com/kubernetes-sigs/cluster-api/pull/190#discussion_r190748203), right?

Reply:
Yes, this was the change I referring to.
The Impl Run needs to be called before starting up the queue workers.